loops_lab.js

```diff
@@ -0,0 +1,197 @@
+//1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+let num = 15
+while (num <= 15 && num > 0){
+  console.log(num);
+  num -=1;
+}
+
+for(let num = 15; num <= 15 && num > 0; num -=1){
```

Comment:
For this loop, you can just do (num > 0). It is usually unnecessary and not recommended practice to put two conditions for loop condition.